tests/introspection/tests.py

```diff
@@ -171,8 +171,8 @@ def test_get_indexes(self):
         with connection.cursor() as cursor:
             indexes = connection.introspection.get_indexes(cursor, Article._meta.db_table)
         if connection.features.can_introspect_index_type:
-            self.assertIn('type', indexes['reporter_id'])
-            indexes['reporter_id'].pop('type')
+            index_type = indexes['reporter_id'].pop('type', None)
+            self.assertEqual(index_type, 'btree')
```

Comment:
I don't think every database on the planet that has `can_introspect_index_type=True` will return "btree" will it? This is really PostgreSQL specific, I thought.